esbuild.config.ts

```diff
@@ -1,17 +1,19 @@
 import esbuild from 'esbuild';
 import { copy } from 'esbuild-plugin-copy';
+import process from 'node:process';
+import url from 'node:url';
 import path from 'node:path';
 // eslint-disable-next-line import/extensions
 import pkg from './package.json';
 
+const dirname = url.fileURLToPath(new URL('.', import.meta.url));
```

Comment:
This bit is to inject `__dirname` into ESM code. A similar pattern can be seen on line 45 where a similar injection needs to be made.

```js
import { createRequire } from 'node:module';
import url from 'node:url';const require = createRequire(import.meta.url);
const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
const __filename = url.fileURLToPath(import.meta.url);
```